### PR TITLE
Update help window

### LINKED ISSUE #219: Update Help Window with the updated commands
The help window was last updated in `v1.2`. So now, it is appropriate that we update the window again for `v1.4`, with the new commands, the updated command formats, etc.

Note that for the help window, we want to include only a command summary so that the user can quickly search for the commands. Else, we can redirect them to the user guide, with which the button to open the UG is already completed.

### PR DESCRIPTION
**Pull request by kevinchangjk**
**Resolves #219**

- Updated help window information
  - Changed to profile commands, advanced management commands, additional commands
  - Resized window (cause sometimes cannot see)
- Minor edits to command summary of UG

### CODE REVIEW

**Comment by kevinchangjk** [comment]
Will fix #166 as well as this is the last of the 3 (UG, feedback, help) to standardise.

**Comment by kevinchangjk** [comment]
Will also fix #194.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -756,19 +758,19 @@ If your changes to the data file makes its format invalid, Coydir will discard a
...
+| **List**            | `list`                                                                                                                                                                               |
+| **Find**            | `find [n/NAME] [j/POSITION] [d/DEPARTMENT]`<br> e.g. `find n/John j/engineer d/Tech`                                                                                                 |
+| **Add leave**       | `add-leave id/ID sd/START_DATE ed/END_DATE`<br> e.g. `add-leave id/1 sd/01-01-2022 ed/02-01-2022`                                                                                    |
+| **Delete leave**    | `delete-leave id/ID i/INDEX`<br> e.g. `delete-leave id/1 i/3`                                                                                                                        |
+| **rate**            | `rate id/ID r/RATING`<br> e.g. `rate id/1 r/3`                                                                                                                                       |
```
Comment (prit3010) [inline docs/UserGuide.md]: Can capitalize rate

**Review by prit3010 — APPROVED** [review]
LGTM

**Pull request merged**